### PR TITLE
11751 add payroll to oca/l10n china

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

**Comment by elicoidal** [comment]
CLA has been fixed (ECLA from Elico Corp)

File: l10n_cn_hr_payroll/README.rst [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+=================
+
+Chinese Payroll Rules
+=====================
+
+    This module extends the functionality of the standard HR Payroll module 
+    and adds complete Chinese payroll rules for the following cities:
+
+    * Shanghai
+    * Shenzhen Are included the following concepts:
```
Comment (elicoidal) [inline l10n_cn_hr_payroll/README.rst]: Add a level in the description

Comment (elicoidal) [inline l10n_cn_hr_payroll/README.rst]: Split the line as followed
- Shenzhen 

Are included the following concepts:

File: l10n_cn_hr_payroll/data/salary_rule_basic.xml [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+            <field name="category_id" ref="hr_payroll.NET"/>
+            <field name="condition_select">none</field>
+            <field name="amount_select">percentage</field>
+            <field name="amount_percentage_base">categories.BASIC + categories.ALW - categories.DED</field>
+            <field name="amount_percentage">100.0</field>
+            <field name="register_id" ref="hr_payroll.contrib_register_employees"/>
+        </record>
+
+        <!-- 津贴 -->
+        <record id="hr_rule_allowance" model="hr.salary.rule">
```
Comment (elicoidal) [inline l10n_cn_hr_payroll/data/salary_rule_basic.xml]: not sure we need this. Is it the last version of the module?

File: l10n_cn_hr_payroll/i18n/l10n_be_hr_payroll.pot [hunk 3]
```diff
@@ -0,0 +1,157 @@
+# Translation of OpenERP Server.
```
Comment (elicoidal) [inline l10n_cn_hr_payroll/i18n/l10n_be_hr_payroll.pot]: we don t need this files

**Comment by Rona111** [comment]
cc @oca-clabot @elicoidal

**Comment by elicoidal** [comment]
LGTM

**Comment by elicoidal** [comment]
close this PR. superseded by #9

**Comment by Rona111** [comment]
superseded by #9

**Pull request closed without merging**